automatic/quiterss/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,25 @@
+$packageArgs = @{
+  packageName   = $env:ChocolateyPackageName
+  softwareName  = 'QuiteRSS'
+  fileType      = 'exe'
+  silentArgs    = "/s /S /q /Q /quiet /silent /SILENT /VERYSILENT /LOG=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).Uninstall.log`""
```

Comment:
don't pass in every silent argument for known installers, only use the ones that actually work for the current installer.

While it may work for some installers, it may not work for others.